docs/content/en/docs-dev/user-guide/adding-an-application.md

```diff
@@ -9,22 +9,17 @@ description: >
 An application is a collect of resources and configurations that are managed together.
 It represents the service which you are going to deploy. With PipeCD, all application's manifests and its application configuration (`app.pipecd.yaml`) must be committed into a directory of a Git repository. That directory is called as application directory.
 
-Before deploying an application, it must be registered via the web console.
-Registering application helps PipeCD know where the application configuration is placed, which `piped` should handle it as well as which cloud the application should be deployed to.
+Before deploying an application, it must be registered via the web console to help PipeCD know
+- where the application configuration is placed
+- which `piped` should handle it and which cloud the application should be deployed to
 
-Each application can be handled by one and only one `piped`. Currently, PipeCD is supporting the following application kinds:
-
-- Kubernetes application
-- Terraform application
-- CloudRun application
-- Lambda application
-- ECS application
+Each application can be handled by one and only one `piped`. Currently, PipeCD is supporting 5 kinds of application: Kubernetes, Terraform, CloudRun, Lambda, ECS.
 
 There are two ways to register an application:
-- Scanning the unused application configuration files in Git to add (recommended)
-- Manually configure application information
+- Picking from a list of unused apps suggested by Pipeds while scanning Git repositories (Recommended way)
+- Manually configuring application information
```

Comment:
nits, how about adding a note/link to mention that we have another way to add bulk of applications via pipectl https://pipecd.dev/docs/user-guide/command-line-tool/#adding-a-new-application

Reply:
As we talked before, that is an edge case where the huge team could face.
But as you said, it might be a good idea to at least mention the link to the page.

Reply:
Sure. Applied!